### PR TITLE
chore: resolve CI problems

### PR DESCRIPTION
**Pull request by salet-gutierrez-jalafund**
## Trello Ticket

[ID-20](https://trello.com/c/oKusHcgV)

## What did I do?

What I did was remove some validations that caused problems when evaluating a pull request, since they were not validated when making a change.
In addition, to evaluate separately, a file was created to evaluate when making a push.

## Why did I do this?

So that the project can be validated correctly and the tasks can be passed without problems since they put everything correctly according to what was indicated.

## How did I do it?

I removed some validations and added that now the commit can be with the Merge branch if the developer performed a merge on his branch.
And so that it can be evaluated on the push I only created a separate file, later I put everything necessary.

### CODE REVIEW

**Review by alex-paca-jalafund — APPROVED** [review]

File: .github/workflows/check-push.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      
+      - name: Build using .Net
+        run: dotnet build
+
+  semantic_pull_request:
```
Comment (joseluis-teran-jalafund) [inline .github/workflows/check-push.yml]: A file related for push events should not have jobs related to PRs events.

**Review by joseluis-teran-jalafund — CHANGES_REQUESTED** [review]

**Review by alex-paca-jalafund — COMMENTED** [review]
There are some related checks that are duplicated I think.

File: .github/workflows/check-pull-request.yml [hunk 2]
```diff
@@ -116,7 +94,7 @@ jobs:
     steps:
       - uses: gsactions/commit-message-checker@v2
         with:
-          pattern: '(?:feat|fix|chore|test|docs|refactor|revert|hotfix)(?:\(\w+\))?:\s[a-z]{1,2}.+(?:\n\nBREAKING CHANGE: .*)?'
+          pattern: '(?:feat|fix|chore|test|docs|refactor|revert|hotfix)(?:\(\w+\))?:\s[a-z]{1,2}.+(?:\n\nBREAKING CHANGE: .*)?|Merge branch .+'
```
Comment (joseluis-teran-jalafund) [inline .github/workflows/check-pull-request.yml:97]: Also, I'd like you add `ci|style` in the pattern. Remember that `feat` convention is related a functionality for the project and use in that is not a good practice. However, you could use `chore` in that case, which is more related.

File: .github/workflows/check-push.yml (hunk 1 above)
Comment (salet-gutierrez-jalafund) [reply .github/workflows/check-push.yml]: done

File: .github/workflows/check-push.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+      
+      - name: Build using .Net
+        run: dotnet build
+
```
Comment (joseluis-teran-jalafund) [inline .github/workflows/check-push.yml:36]: Also, I recommend you to write a job for the branches which will go back to develop, like the one which is on `check-pull-request.yml` file

**Review by joseluis-teran-jalafund — CHANGES_REQUESTED** [review]

File: .github/workflows/check-pull-request.yml (hunk 2 above)
Comment (salet-gutierrez-jalafund) [reply .github/workflows/check-pull-request.yml:97]: In the wiki, in the commits section, the `ci|style` is not mentioned, that is why I did not enter it, but you have zaron when it comes to adding a feat, I would have to enter a chore, that is the best, thank you very much for the recommendation.
If we would like to increase the `ci|style` please let me know if we would have to modify the methodologies.

File: .github/workflows/check-push.yml (hunk 3 above)
Comment (salet-gutierrez-jalafund) [reply .github/workflows/check-push.yml:36]: Done

**Review by joseluis-teran-jalafund — APPROVED** [review]
Good job!

**Comment by salet-gutierrez-jalafund** [comment]
[ID-20: Set Up CI Workflows](https://trello.com/c/oKusHcgV/28-id-20-set-up-ci-workflows)

**Pull request merged**